blockchain/blockchain.go

```diff
@@ -248,6 +252,7 @@ func NewBlockChain(db database.DBManager, cacheConfig *CacheConfig, chainConfig
 		db:                 db,
 		triegc:             prque.New(),
 		chBlock:            make(chan gcBlock, 1000),
+		chPrune:            make(chan uint64, 1000),
```

Comment:
The default block chunk size from downloader is 2048. If the channel size is 2048, this channel will not be blocked during the block chunk processing.

Reply:
Both `chBlock` and `chPrune` are used when not in archive mode, meaning the channel is stuffed every other 128 blocks. But I will increase them to 2048 anyway because that's not too big.